docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -276,39 +274,108 @@ You can specify multiple database filters by adding additional list elements.
       regions: ["eu-west-1"]
       tags:
         "*": "*"
+  annotations:
+    serviceAccount:
+      eks.amazonaws.com/role-arn: arn:aws:iam::1234567890:role/my-rds-autodiscovery-role
   ```
 
-  <Admonition type="note" title="YAML formatting">
-    YAML is very sensitive to correct spacing. When specifying lists in a `values.yaml` file, you might like
-    to [use a linter](https://codebeautify.org/yaml-validator) to validate your YAML list and ensure that it is correctly formatted.
-  </Admonition>
+  (!docs/pages/includes/yaml-lint-note.mdx!)
   </TabItem>
-  <TabItem label="--set">
```

Comment:
Why remove the `--set` version of this and the other edited sections?

Reply:
The `--set` tab was extremely cumbersome to use for config - it seems misleading to even suggest someone use that method for configuration. Hugo suggested removing it and I agreed that no one is realistically going to bother with --set here. I don't care if we bring it back though, so it's up to you